.github/workflows/ci.yml

```
@@ -1,86 +1,347 @@
+            fi
+            echo "Waiting... ($i/30)"
+            sleep 1
+          done
+
       - name: Run OWASP ZAP Baseline Scan
-        uses: zaproxy/action-baseline@v0.10.0
+        uses: zaproxy/action-baseline@v0.12.0
         with:
           target: 'http://localhost:8000'
+          rules_file_name: '.zap/rules.tsv'
+          cmd_options: '-a' # Ajax spider
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)